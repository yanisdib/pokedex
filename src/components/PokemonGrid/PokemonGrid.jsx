import styled from 'styled-components';


export default styled.div`
    /* Landscape phones and down */
    @media (min-width: 320px) { 
        display: grid;
        grid-template-columns: 1fr 1fr;
        column-gap: 10px;
        row-gap: 30px;
        max-width: 100%;
        width: 100%;
        align-content: center;
        justify-items: center;
        margin: 30px 0; 
    }

    /* Landscape phone to portrait tablet */
    @media (max-width: 480px) { 
        display: grid;
        grid-template-columns: 1fr 1fr;
        column-gap: 10px;
        row-gap: 30px;
        max-width: 100%;
        align-content: center;
        justify-items: center;
        margin: 30px 0; 
    }

    /* Portrait tablet to landscape and desktop */
    @media (min-width: 768px) { 
        display: grid;
        grid-template-columns: 1fr 1fr 1fr;
        column-gap: 20px;
        row-gap: 30px;
        max-width: 100%;
        align-content: center;
        justify-items: center;
        margin: 30px 0; 
    }

    /* Large desktop */
    @media (min-width: 1200px) { 
        display: grid;
        grid-template-columns: 1fr 1fr 1fr;
        column-gap: 20px;
        row-gap: 30px;
        max-width: 800px;
        align-content: center;
        justify-items: center;
        margin: 30px 0; 
    }
`;