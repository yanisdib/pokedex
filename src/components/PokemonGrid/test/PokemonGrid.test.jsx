import { shallow } from 'enzyme';

import PokemonGrid from '../PokemonGrid';


describe('PokemonGrid component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<PokemonGrid />))

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})