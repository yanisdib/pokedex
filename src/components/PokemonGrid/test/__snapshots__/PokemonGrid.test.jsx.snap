// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonGrid component renders correctly without crashing 1`] = `ShallowWrapper {}`;
